test/unit/conversations_page_test.dart

```diff
@@ -31,7 +31,7 @@ void main() {
       expect(inject_list, alternate_list);
     });
 
-    test('.populateWith() does not replace _items when pre initialised', () {
+    test('.populateWith() replaces _items when already has items', () {
```

Comment:
It great that writing tests is helping us remove quirks from the system with a more robust approach. Previously I had to assume there was a purpose to why it was implemented in such a way.

Reply:
I'm loving how much testing we're doing! We're approaching rockstar levels ;-)